db/penalty_test.cc

```diff
@@ -108,11 +108,20 @@ TEST(PenaltyTester, Debug1)
     
     UpdatePenalty(&version);
 
-    ASSERT_EQ(version.WritePenalty(), 0);
+    ASSERT_TRUE(0==version.WritePenalty());
 
 }   // test Debug1
 #endif
 
+  /*******************
```

Comment:
I recommend moving this comment block above the first test (Debug1) since it applies to that test, too (even though it is currently commented out).